packages/@o3r/storybook/project.json

```diff
@@ -50,15 +50,22 @@
       "options": {
         "script": "build:wrapper"
       },
-      "dependsOn": ["^build", "compile"]
```

Comment:
are you sure it's not needed ? there seems to be at least a dependency on `@o3r/configuration`

Reply:
`compile` is already depending on `^build`

Reply:
fair enough